libs/components/src/lib/login-form/login-form.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as yup from 'yup';
+import {yupResolver} from '@hookform/resolvers/yup';
```

Comment:
some rules are missed it should be `import { yupResolver } ...`

Reply:
rules were added